pybamm/util.py

```diff
@@ -356,6 +367,11 @@ def install_jax():
 
     if system() == "Windows":
         raise NotImplementedError("Jax is not available on Windows")
+    elif importlib.util.find_spec("jax") is not None:
+        if not is_jax_compatible():
+            raise ValueError(
```

Comment:
Codecov shows that this complete function is not included in the coverage, should I add `# pragma: no cover` for this function?

Reply:
That sounds good

Reply:
Also, this is now an error. Should we make it a warning and proceed without `jax`?

Reply:
PyBaMM installation (and importing PyBaMM) proceeds without `jax` if the installed `jax` version is not compatible. But this function throws an error if a user tries to install `jax` and has an incompatible `jax` version already installed.

Should it instead uninstall the existing version and install the compatible version?

Reply:
Sorry, I misunderstood! That works as it is :)